app/components/GettingStartedModal/GettingStartedModal.tsx

```diff
@@ -0,0 +1,257 @@
+    const handleTechnologiesStepChange = (selectedTechnologies: GettingStartedTechnologiesData) => {
+        setTechnologiesStepData(selectedTechnologies);
+    };
+    const handleSecretsStepChange = (typedSecrets: GettingStartedSecretsData) => {
+        setSecretsStepsData({ ...secretsStepsData, [secretsStepSchema.name]: typedSecrets });
+    };
+    const handleInstallationStarted = () => {
+        setInstallationProcessing(true);
+    };
+    const handleInstallationFinishedOrCanceled = () => {
+        EventBus.trigger('plugins:refresh');
+        EventBus.trigger('secrets:refresh');
+        setInstallationProcessing(false);
+    };
+    const handleModalClose = () => {
+        setModalOpen(false);
+        if (modalDisabledChecked) {
+            setGettingStartedModalDisabled(true);
+        }
+    };
+
+    const handleBackClick = () => {
+        switch (stepName) {
+            case StepName.Status:
+                setStepName(StepName.Summary);
+                break;
+
+            case StepName.Summary:
+                if (secretsStepsSchemas.length > 0) {
+                    setStepName(StepName.Secrets);
+                    setSecretsStepIndex(secretsStepsSchemas.length - 1);
+                } else {
+                    setStepName(StepName.Technologies);
+                }
+                break;
+
+            case StepName.Secrets:
+                setStepErrors([]);
+                if (secretsStepIndex > 0) {
+                    setSecretsStepIndex(secretsStepIndex - 1);
+                } else {
+                    setStepName(StepName.Technologies);
+                }
+                break;
+
+            default:
+                log.error('Incorrect step name.');
+                break;
+        }
+    };
+    const handleNextClick = () => {
+        switch (stepName) {
+            case StepName.Technologies:
+                if (checkTechnologiesStepDataErrors()) {
+                    setStepName(StepName.Secrets);
+                    setSecretsStepIndex(0);
+                }
+                break;
+
+            case StepName.Secrets:
+                if (checkSecretsStepDataErrors()) {
+                    if (secretsStepIndex < secretsStepsSchemas.length - 1) {
+                        setSecretsStepIndex(secretsStepIndex + 1);
+                    } else {
+                        setStepName(StepName.Summary);
+                    }
+                }
+                break;
+
+            case StepName.Summary:
+                setStepName(StepName.Status);
+                break;
+
+            default:
+                log.error('Incorrect step name.');
+                break;
+        }
+    };
+
+    return (
+        <Modal open={modalOpen} onClose={handleModalClose}>
+            <Modal.Header>{getHeaderText(secretsStepsSchemas, stepName, secretsStepIndex)}</Modal.Header>
+            <Modal.Content style={{ minHeight: 220 }}>
+                {stepErrors && stepErrors.length > 0 && (
```

Comment:
Maybe we can use:
```suggestion
                {!_.isEmpty(stepErrors) && (
```
here?

Reply:
fixed